src/foam/java/PropertyInfo.js

```diff
@@ -64,6 +77,18 @@ foam.CLASS({
             type: 'String',
             body: 'return "' + this.propName + '";'
           },
+          {
+            name: 'getShortName',
+            visibility: 'public',
+            type: 'String',
+            body:  this.propShortName ? 'return "' +this.propShortName+'";' : 'return null;'
+          },
+          {
+            name: 'getAliases',
+            visibility: 'public',
+            type: 'String[]',
+            body: 'return ' + this.buildBody
```

Comment:
"buildBody" is a pretty generic name.  We only appear to use buildBody in this spot.  Can we just inline the buildBody expression here?